packages/core/src/Provider/HandleProvider/types.ts

```diff
@@ -0,0 +1,23 @@
+import { ByronAddress, EnterpriseAddress, PaymentAddress, PointerAddress } from '../../Cardano/Address';
+import { Point } from '../..';
+
+export type Handle = string;
+
+type CardanoAddress = ByronAddress | PaymentAddress | EnterpriseAddress | PointerAddress;
+
+export interface HandleInfo {
```

Comment:
Since it has `resolvedAt`, it's represents a handle resolved at some point in time - I think `HandleResolution` would communicate the purpose of the type a little better. Also, we would likely have more handle issuers in the future, so it would be great to add an `issuer` property, to not implicitly couple this with 1 implementation.

```suggestion
export enum HandleIssuer {
  KoraLabs = 'KoraLabs'
}

export interface HandleResolution {
  issuer: HandleIssuer;
```